charts/alertmanager/values.yaml

```diff
@@ -101,3 +104,5 @@ config:
     group_interval: 5m
     receiver: default-receiver
     repeat_interval: 3h
+
+template: |-
```

Comment:
imho it would be better to do this like:

```
templates:
  - name: "/etc/alertmanager/alertmanager.tmpl"
    content: ""
```

Otherwise its unclear, that these options belong together.

Reply:
would `name: alertmanager.tmpl` suffice? it would go in same dir as other config i imagine (/etc/alertmanager)

Reply:
yes :)

Reply:
I'm afraid I don't get your point. We are dumping `config` as a whole to the `alertmanager.yml` file, so altering `templates` config key structure will not be accepted by Alertmanager as it [expects](https://prometheus.io/docs/alerting/latest/configuration/#filepath) to see simple list of files, not list of maps:

![Screenshot 2020-12-08 at 17 52 32](https://user-images.githubusercontent.com/31690776/101506528-29824d80-397e-11eb-844e-537d31e3725b.png)